### PR TITLE
Pmd findbugs

### LINKED ISSUE #78: PMD and Findbugs
There are a lot of PMD and Findbugs warnings that have to be looked at, and if reasonable, solved

### PR DESCRIPTION
**Pull request by mennoo1996**
Fixes #78 
Fixed PMD and findbugs warnings. Two types of PMD warnings remain
Unused private field for fields that are not used in the class itself but
only via an @Getter
Unused method parameter event (MouseEvent mostly) because of usage of
lambda's and the mouse event itself is not actually used

### CODE REVIEW

File: src/main/java/control/ToolViewController.java [hunk 1]
```diff
@@ -36,10 +34,10 @@ public ToolViewController(ControllerManager controllerManager) {
      * the event handlers.
      */
     private void initializeTools() {
-        cameraBlockCreationTool = new ToolButton("Add camerashot",
+        new ToolButton("Add camerashot",
```
Comment (bart-de-jonge) [inline src/main/java/control/ToolViewController.java:37]: are you sure we don't need these fields in the future?

Comment (mennoo1996) [inline src/main/java/control/ToolViewController.java:37]: If they are needed in the future we can add them in the future. At this moment, it is bad practice to have unused variables. Unless someone is going to tell me that they are using these variables at this very moment while implementing new features

File: src/main/java/gui/modal/CameraShotCreationModalView.java [hunk 2]
```diff
@@ -75,11 +73,11 @@
 
     // No touching these constants. They work well for all general cases,
     // and there is no reason to change them ever again.
-    private final int generalSize = 10000;
-    private final int generalSpacing = 10;
-    private final int generalPadding = 20;
-    private final int textAreaMinWidth = 350;
-    private final int cameraAreaMinWidth = 250;
+    private static final int generalSize = 10000;
+    private static final int generalSpacing = 10;
```
Comment (bart-de-jonge) [inline src/main/java/gui/modal/CameraShotCreationModalView.java]: constants should be IN_CAPITAL_LETTERS_ONLY. can you change this? Should be easy with the refactor function of the ide

Comment (mennoo1996) [inline src/main/java/gui/modal/CameraShotCreationModalView.java]: Will do

**Comment by bart-de-jonge** [comment]
Two small comments, other than that great work. Keeping the checkers satisfied keeps our code clean :)

File: src/main/java/control/ToolViewController.java (hunk 1 above)
Comment (bart-de-jonge) [inline src/main/java/control/ToolViewController.java:37]: Fair point, I was just checking. Deleting them seems like the right thing to do.

**Pull request merged**